sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/AsynReadWithMultipleClients.java

```diff
@@ -0,0 +1,299 @@
+                        .buildAsyncClient();
+                    List<PojoizedJson> docsToRead = new ArrayList<>();
+                    if (!configuration.isDeleteCollections()) {
+                        CosmosAsyncDatabase cosmosAsyncDatabase = asyncClient.createDatabaseIfNotExists(this.configuration.getDatabaseId()).block().getDatabase();
+                        CosmosAsyncContainer cosmosAsyncContainer = cosmosAsyncDatabase.createContainerIfNotExists(configuration.getCollectionId(), PARTITION_KEY, 100000).block().getContainer();
+                        String partitionKey = cosmosAsyncContainer.read().block().getProperties().getPartitionKeyDefinition()
+                            .getPaths().iterator().next().split("/")[1];
+                        String dataFieldValue = RandomStringUtils.randomAlphabetic(this.configuration.getDocumentDataFieldSize());
+                        ArrayList<Flux<PojoizedJson>> createDocumentObservables = new ArrayList<>();
+
+                        for (int i = 0; i < this.configuration.getNumberOfPreCreatedDocuments(); i++) {
+                            String uuid = UUID.randomUUID().toString();
+                            com.azure.cosmos.benchmark.PojoizedJson newDoc = generateDocument(uuid, dataFieldValue, partitionKey);
+
+                            Flux<PojoizedJson> obs = cosmosAsyncContainer.createItem(newDoc).map(resp -> {
+                                    com.azure.cosmos.benchmark.PojoizedJson x =
+                                        resp.getItem();
+                                    return x;
+                                }
+                            ).flux();
+                            createDocumentObservables.add(obs);
+                        }
+                        docsToRead = Flux.merge(Flux.fromIterable(createDocumentObservables), 100).collectList().block();
+                        MeterRegistry registry = configuration.getAzureMonitorMeterRegistry();
+
+                        if (registry != null) {
+                            BridgeInternal.monitorTelemetry(registry);
+                        }
+
+                        registry = configuration.getGraphiteMeterRegistry();
+
+                        if (registry != null) {
+                            BridgeInternal.monitorTelemetry(registry);
+                        }
+                        System.out.println("Client have been initialized with data created for host " + hostAndKey[0]);
+                    } else {
+                        System.out.println("Client have been initialized with host " + hostAndKey[0]);
+                    }
+                    clientDocsMap.put(asyncClient, docsToRead);
+                }
+            }
+            System.out.println("Total number of client created for ReadThroughputWithMultipleClient " + clientDocsMap.size());
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```

Comment:
please don't use System.out/err.
use logger here and elsewhere.

Reply:
done

Reply:
you still have e.printStackTrace() throughout the file. please fix everywhere.